lib/inputstreamhelper/__init__.py

```diff
@@ -959,3 +996,19 @@ def info_dialog(self):
         text += localize(30830, url=config.ISSUE_URL)  # Report issues
 
         Dialog().textviewer(localize(30901), text)
+
+    def rollback_libwv(self):
+        """Backup current lib and rollback to a version specified by the user"""
+        backup_path = self._backup_path()
+        versions = os.listdir(backup_path)
+        show_versions = []
+
+        for version in versions:
+            lib_version = self._get_lib_version(os.path.join(backup_path, version, config.WIDEVINE_CDM_FILENAME[system_os()]))
```

Comment:
Shouldn't we rely completely on container version and only read the manifest.json?

Reply:
On ARM this "container version" would be the version of the ChromeOS image and we don't have a manifest.json there. I think it makes sense there at least. If you want to I could remove this on x86.

Reply:
I think it's better to remove this for x86 because the versions in https://dl.google.com/widevine-cdm/versions.txt only match with the manifest.json
I tested and 4.10.1440.19 gets never backed up because the lib version matches 4.10.1440.18

Reply:
Alright, will change it in a minute (though I always prefer to keep it platform independent).
I don't understand why 4.10.1440.19 would not be backed up, since `_get_lib_version()` is never called during the backup process. During my tests it worked.